src/native-select/index.tsx

```diff
@@ -121,7 +131,7 @@ export const NativeSelect = factory(function NativeSelect({
 					}}
 					classes={themedCss.select}
 				>
-					{!initialValue && <option key="blank-option" value="" />}
+					{value === undefined && <option key="blank-option" value="" />}
```

Comment:
This changes the behavior. Previously is no `initialValue` was provided then this blank value would always display, even after selecting a value. Now it is only displayed until you select a value and then goes away.

Reply:
fair point, please can you fix this @ericos